### ISSUE TITLE
Registry pods are not removed when the plan is deleted

### ISSUE THREAD

**Issue by sergiordlr** [fusor/mig-ui#397]
Hello,

After creating a migration plan and deleting it, the register pods in OCP4 are not deleted.

```
oc get pods -n mig
NAME                                  READY   STATUS      RESTARTS   AGE
controller-manager-6b999485db-x4s67   1/1     Running     0          92m
gfrwx-registry-2tnbm-1-b6bjc          1/1     Running     0          20m
gfrwx-registry-2tnbm-1-deploy         0/1     Completed   0          20m
migration-operator-bfbc7f445-5lb6w    2/2     Running     0          93m
migration-ui-8d7d66bb-pcf7z           1/1     Running     0          92m
nfsmove-registry-fjhfj-1-deploy       0/1     Completed   0          10m
nfsmove-registry-fjhfj-1-djsf6        1/1     Running     0          10m
restic-jqv8f                          1/1     Running     0          93m
restic-swhsz                          1/1     Running     0          93m
restic-z7859                          1/1     Running     0          93m
rwoplan-registry-pt7gb-1-deploy       0/1     OOMKilled   0          60m
rwoplan-registry-pt7gb-1-lfwtf        1/1     Running     0          60m
rwxplan-registry-trr94-1-7z728        1/1     Running     0          54m
rwxplan-registry-trr94-1-deploy       0/1     Completed   0          54m
velero-6bf58f4f88-scpd7               1/1     Running     0          93m
```
To reproduce it, create a plan, wait until the regristy pod is created, delete the plan. You will see that the registry pod is still there after deletion.

After talking with John Matthews, the problem seems to be related to not closing the plan when deleted.

Best regads,

Thank you.

**Comment by eriknelson** [comment]
This is related to the UI not closing plans prior to delete. Need to determine what our desired short term behavior is here.

**Comment by eriknelson** [comment]
Fixed by #436
